examples/http-cookies/cookie_server.bal

```diff
@@ -4,65 +4,54 @@ listener http:Listener serverEP = new (9095);
-                    // path as `/` to apply it to all the resources in the service.
-                    http:Cookie cookie = new("username", name.toString(),
-                                                path = "/");
+            // [Create a new cookie](https://docs.central.ballerina.io/ballerina/http/latest/classes/Cookie)
+            // by setting `name` as the `username` and `value` as the logged-in user's name. Set the cookies
+            // path as `/` to apply it to all the resources in the service.
+            http:Cookie cookie = new("username", name.toString(), path = "/");
 
-                    http:Response response = new;
+            http:Response response = new;
 
-                    // [Add the created cookie to the response](https://docs.central.ballerina.io/ballerina/http/latest/classes/Response#addCookie).
-                    response.addCookie(cookie);
+            // [Add the created cookie to the response](https://docs.central.ballerina.io/ballerina/http/latest/classes/Response#addCookie).
+            response.addCookie(cookie);
 
-                    // Set a message payload to inform that the login has
-                    // been succeeded.
-                    response.setTextPayload("Login succeeded");
-                    return response;
-                }
-            }
+            // Set a message payload to inform that the login has
+            // been succeeded.
+            response.setTextPayload("Login succeeded");
+            return response;
         }
-        return {body: "Invalid request payload"};
+        return http:UNAUTHORIZED;
     }
 
     resource function get welcome(http:Request req) returns string {
         // [Retrieve cookies from the request](https://docs.central.ballerina.io/ballerina/http/latest/classes/Request#getCookies).
         http:Cookie[] cookies = req.getCookies();
 
         // Get the cookie value of the `username`.
-        http:Cookie[] usernameCookie = cookies.filter(function
-                                (http:Cookie cookie) returns boolean {
+        http:Cookie[] usernameCookie = cookies.filter(function(http:Cookie cookie) returns boolean {
             return cookie.name == "username";
         });
 
-        if (usernameCookie.length() > 0) {
+        if usernameCookie.length() > 0 {
             string? user = usernameCookie[0].value;
-            if (user is string) {
+            if user is string {
                 // Respond with the username added to the welcome message.
                 return "Welcome back " + user;
 
-            } else {
-                // If the user is `nil`, send a login message.
-                return "Please login";
             }
-        } else {
-            // If the `username` cookie is not presented, send a login message.
+            // If the user is `nil`, send a login message.
             return "Please login";
+
         }
+        // If the `username` cookie is not presented, send a login message.
```

Comment:
```suggestion
        // If the `username` cookie is not presented, send a login message.
```
What is a login message? May be, it can be described more.